code/modules/client/preference_setup/loadout/loadout_suit.dm

```diff
@@ -161,65 +157,32 @@
 	coat["technicolor trenchcoat (Detective)"] = /obj/item/clothing/suit/storage/toggle/det_trench/technicolor
 	gear_tweaks += new/datum/gear_tweak/path(coat)
 
+/datum/gear/suit/winter
+	display_name = "winter coat selection"
+	description = "A selection of coats for the thermally challenged."
+	path = /obj/item/clothing/suit/storage/hooded/wintercoat
+
+/datum/gear/suit/winter/New()
+	..()
+	var/wintercoat = list()
+	wintercoat["winter coat"] = /obj/item/clothing/suit/storage/hooded/wintercoat
+	wintercoat["winter coat, red"] = /obj/item/clothing/suit/storage/hooded/wintercoat/red
+	wintercoat["winter coat, captain"] = /obj/item/clothing/suit/storage/hooded/wintercoat/captain
+	wintercoat["winter coat, security"] = /obj/item/clothing/suit/storage/hooded/wintercoat/security
+	wintercoat["winter coat, science"] = /obj/item/clothing/suit/storage/hooded/wintercoat/science
+	wintercoat["winter coat, IAC"] = /obj/item/clothing/suit/storage/hooded/wintercoat/iac
+	wintercoat["winter coat, medical"] = /obj/item/clothing/suit/storage/hooded/wintercoat/medical
+	wintercoat["winter coat, engineering"] = /obj/item/clothing/suit/storage/hooded/wintercoat/engineering
+	wintercoat["winter coat, hydroponics"] = /obj/item/clothing/suit/storage/hooded/wintercoat/hydro
+	wintercoat["winter coat, cargo"] = /obj/item/clothing/suit/storage/hooded/wintercoat/cargo
+	wintercoat["winter coat, mining"] = /obj/item/clothing/suit/storage/hooded/wintercoat/cargo
```

Comment:
```suggestion
	wintercoat["winter coat, mining"] = /obj/item/clothing/suit/storage/hooded/wintercoat/mining
```

Reply:
Also, I don't see the point of removing the atmospherics one, when it's a well liked color scheme.

Reply:
ditto